kingsleyreadlexicon.tsv

```diff
@@ -4651,7 +4651,7 @@ athenian	𐑩𐑔𐑰𐑯𐑾𐑯	AJ0	əˈθiːnɪən	295
 athenian	𐑩𐑔𐑰𐑯𐑾𐑯	NN1	əˈθiːnɪən	1
 athenians	𐑩𐑔𐑰𐑯𐑾𐑯𐑟	NN2	əˈθiːnɪənz	148
 athens	𐑨𐑔𐑩𐑯𐑟	NP0	ˈæθ.ənz	1039
-atherstone	𐑨𐑔𐑼𐑕𐑑𐑩𐑯	NP0	ˈæθ.əR.stəʊn	14
+atherstone	𐑨𐑔𐑼𐑕𐑑𐑩𐑯	NP0	ˈæθ.əR.stən	14
```

Comment:
To follow the first listed pronunciation in the Cambridge Pronouncing Dictionary I suggest:

`atherstone	𐑨𐑔𐑼𐑕𐑑𐑴𐑯	NP0	ˈæθ.əR.stəʊn	14`